proposals/intptr-operators.md

```diff
@@ -30,7 +30,245 @@ As such, the language should begin to support these operators to help advance th
 ## Detailed design
 [design]: #detailed-design
 
-The full set of operators supported are defined in `III.1.5` of the Common Language Infrastructure specification (`ECMA-335`). The specification is available here: [https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf)
+The full set of operators supported are defined in `III.1.5` of the Common Language Infrastructure specification (`ECMA-335`). The specification is available here: [https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf).
+* A summary of the operators is provided below for convenience.
+* The unverifiable operators defined by the CLI spec are not listed and are not currently part of this proposal (although it may be worth considering these as well).
+* Providing a keyword (such as `nint` and `nuint`) nor providing a way to for literals to be declared for `System.IntPtr` and `System.UIntPtr` (such as 0n) is part of this proposal (although it may be worth considering these as well).
+
+#### Unary Plus Operator
+
+```C#
+System.IntPtr operator +(System.IntPtr)
+```
+
+```C#
+System.UIntPtr operator +(System.UIntPtr)
+```
+
+#### Unary Minus Operator
+
+```C#
+System.IntPtr operator -(System.IntPtr)
+```
+
+#### Bitwise Complement Operator
+
+```C#
+System.IntPtr operator ~(System.IntPtr)
+```
+
+```C#
+System.UIntPtr operator ~(System.UIntPtr)
+```
+
+#### Cast Operators
+
+```C#
+operator sbyte(System.IntPtr)               // Truncate
```

Comment:
Explicit conversions are not specified as methods (if it were, these would need to be `explicit` oeprators), but are rather specified as changes to the spec section on explicit conversions. But this is fine for now.